CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 1.7.0 / 2017-06-06
+
+* [CHANGE] Compress remote storage requests and responses.
```

Comment:
I see, I didn't follow entirely so I just used the PR title. What would you suggest?